cli/code_scanner.py

```diff
@@ -391,6 +423,48 @@ def exclude_irrelevant_scan_results(document_detections_list: List[DocumentDetec
     return relevant_document_detections_list
 
 
+def parse_pre_receive_input() -> str:
+    pre_receive_input = sys.stdin.read().strip()
+    if not pre_receive_input:
+        raise ValueError("pre receive input was not found")
```

Comment:
Make sure that you catch this error and print something like "use this command to scan in git pre-recive hook only"